api/app/Models/Classification.php

```diff
@@ -46,9 +47,23 @@ public function genericJobTitles(): HasMany
      */
     protected function displayName(): Attribute
     {
+        /** @disregard P1003 Not using values */
         return Attribute::make(
             get: fn (mixed $value, array $attributes) => $attributes['group'].'-'.sprintf('%02d', $attributes['level']),
 
         );
     }
+
+    public static function scopeAvailableInSearch(Builder $query, bool $availableInSearch)
```

Comment:
Let's add a comment explaining why this is here and what its for. Also, add a TODO to remove this in [#9483](https://github.com/GCTC-NTGC/gc-digital-talent/issues/9483).